cwf/gateway/docker/python/Dockerfile

```diff
@@ -151,3 +151,8 @@ WORKDIR /
 
 # Copy the build artifacts.
 COPY --from=builder /build/python/gen /usr/local/lib/python3.5/dist-packages/
+
+RUN rm -rf /usr/src/linux-headers* /ovs/
+
+FROM scratch as lte_gateway_python
+COPY --from=prod_preimage / /
```

Comment:
This looks hella sketchy, what's the point of this copy? Why not just use prod_preimage as-is and call it lte_gateway_python?